src/main/python/RLBotFramework/gui/base_gui.py

```diff
@@ -0,0 +1,149 @@
+import os
+from PyQt5.QtCore import QTimer
+
+from RLBotFramework.gui.index_manager import IndexManager
+from RLBotFramework.gui.base_gui_agent import BaseGuiAgent
+from RLBotFramework.parsing.rlbot_config_parser import create_bot_config_layout, get_num_players
+from RLBotFramework.gui.presets import LoadoutPreset, AgentPreset
+from RLBotFramework.parsing.agent_config_parser import PARTICIPANT_CONFIGURATION_HEADER, PARTICIPANT_LOADOUT_CONFIG_KEY
+from RLBotFramework.agents.base_agent import BOT_CONFIG_MODULE_HEADER, LOOKS_CONFIG_KEY
+from RLBotFramework.utils.class_importer import get_base_repo_path
```

Comment:
You've attempted this import in a few places, but get_base_repo_path does not exist anymore in class_importer.py. Seems like the v4 merge did not go perfectly and it broke your code.